src/driver/postgres/PostgresDriver.ts

```diff
@@ -1084,6 +1085,7 @@ export class PostgresDriver implements Driver {
      */
     protected executeQuery(connection: any, query: string) {
         return new Promise((ok, fail) => {
+            this.connection.logger.logQuery(query);
```

Comment:
@imnotjames I'm unsure if this is the right way to do this or not

Reply:
Yeah, it probably should have been doing that anyway